model/src/pyrenew/latent/hospitaladmissions.py

```diff
@@ -22,13 +21,18 @@ class HospitalAdmissionsSample(NamedTuple):
         The infection-to-hospitalization rate. Defaults to None.
     latent_hospital_admissions : SampledValue or None
         The computed number of hospital admissions. Defaults to None.
+    day_of_week_effect_mult : SampledValue or None
```

Comment:
I think it makes more sense to just call this `multiplier`. It's not really specific to the day of the week effect.

Reply:
Sure